tests/test_apps/upgraded/RectorCommand-testApply53/src/SomeTest.php

```diff
@@ -27,4 +29,21 @@ public function testRenames(): void
     public function findSomething(\Cake\ORM\Query\SelectQuery $query, array $options): \Cake\ORM\Query\SelectQuery {
         return $query;
     }
+
+    public function routes(): void
+    {
+        $routes = new RouteBuilder(new RouteCollection(), '/');
+
+        $routes->scope(path: '/api', params: [], callback: function ($routes): void {});
+        $routes->scope(path: '/api', params: ['something'], callback: function ($routes): void {});
+
+        $routes->resources(name: '/api', callback: function ($routes): void {});
+        $routes->resources(name: '/api', options: ['something'], callback: function ($routes): void {});
```

Comment:
```suggestion
        $routes->resources('/api', options: ['something'], callback: function ($routes): void {});
```

Is there a way to not use a named parameter here? This is arguably a style choice and feel free to not make any changes.

Reply:
It is definitely allowed and possible to do so. And in this case I do also like this approach of a mixed non-named and named arguments approach.